.github/renovate.json

```diff
@@ -16,11 +16,17 @@
       "automerge": true
     },
     {
-      "description": "Group container base image updates",
+      "description": "Pin Fedora to stable version 42, block pre-release versions",
+      "matchPackagePatterns": ["registry.fedoraproject.org/fedora"],
+      "allowedVersions": "/^42(\\.|$)/",
```

Comment:
This will become outdated in about 4 months, we need something more robust to figure out which Fedora release is the latest stable one.